analytics/pyproject.toml

```diff
@@ -76,7 +76,6 @@ ignore = [
   "D203",    # no blank line before class
   "D212",    # multi-line summary first line
   "FIX002",  # line contains TODO
-  "PD901",   # pandas df variable name
```

Comment:
Ruff was raining a warning that this [rule was deprecated](https://docs.astral.sh/ruff/rules/pandas-df-variable-name/#pandas-df-variable-name-pd901)